.travis.yml

```diff
@@ -51,6 +54,9 @@ install:
 script:
   - cd ${TRAVIS_BUILD_DIR}
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then ulimit -S -n 2048; fi
+  - if [[ $SUFFIX_TEST == 'true' ]]; then
```

Comment:
this if is messy because the extra test config is defined in 3? different places across the yaml.  Could it work so that the extended `${setup_cmd}` is defined in the `matrix:` section only?  Ideally something like `setup_cmd = ${setup_cmd} + '--no-append-suffix'`

Reply:
Makes sense. I'll clean it up.